### ISSUE TITLE
gotcha

### ISSUE THREAD

**Issue by RobinHankin** [RobinHankin/hyper2#82]
Function `loglik()` ignores the names of a named vector:

``` r
suppressMessages(library("hyper2"))
(H <- hyper2(list("a",c("a","b"),c("b","c")),1:3))
#> Warning in print.hyper2(x): powers have nonzero sum
#> log( a * (a + b)^2 * (b + c)^3)
loglik(c(a=0.2,b=0.6),H)
#> [1] -2.725156
loglik(c(b=0.6,a=0.2),H) # should be identical (?)
#> [1] -3.705985
```

<sup>Created on 2023-06-26 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

one might expect that the two calls to `loglik()` would result in the same likelihood, because they arguably refer to the same point in probability space.  But `loglik()` ignores the names of a named vector.  It might be good to document this.

**Comment by RobinHankin** [comment]
it might be a better idea to figure out the desired behaviour first, then implement it, then document it.

**Comment by RobinHankin** [comment]
Fixed in 594c316
